themes/default/layouts/page/pulumi-up.html

```diff
@@ -136,5 +84,30 @@ <h3 class="text-center">Supported by our friends at</h3>
         </ul>
     </section>
 
+    <section id="get-started" class="container px-6 lg:px-0 mx-auto my-16">
+        <div class="w-full bg-violet-600 card p-6 lg:p-16 lg:pt-24 text-center">
+            <div class="max-w-xl mx-auto">
+                <h2 class="text-white hidden lg:block px-0 lg:px-16">
+                    Get started with Cloud Engineering
+                </h2>
+                <h4 class="text-white mt-0 lg:hidden">
+                    Pulumi is a cloud engineering platform that brings infrastructure, application development,
+                    and compliance teams together through a unified software engineering process to deliver
+                    and manage modern cloud applications.
+                </h4>
+                <p class="text-white">
+                    Pulumi is a cloud engineering platform that brings infrastructure, application development,
+                    and compliance teams together through a unified software engineering process to deliver
+                    and manage modern cloud applications.
+                </p>
```

Comment:
On mobile this renders as repeated content:

![image](https://user-images.githubusercontent.com/274700/118728868-b660eb00-b7e9-11eb-995d-6920eff72851.png)

I think it should probably be something like

```suggestion
                <p class="text-white text-2xl lg:text-base font-display lg:font-body">
                    Pulumi is a cloud engineering platform that brings infrastructure, application development,
                    and compliance teams together through a unified software engineering process to deliver
                    and manage modern cloud applications.
                </p>
```

Reply:
🤦 Thanks for catching this. The h4 header is just the mobile version of the section header.